### PR TITLE
Misc viewer related fixes

### PR DESCRIPTION
**Pull request by ryantrem**
Three small fixes:
1. When an environment fails to load, roll back any related state mutations (especially scene autoclear).
2. Add a 1px border to the default progress bar since by default the background will typically be white, which makes the progress bar hard to see. Users can always apply custom styles to the progress bar as well.
3. Dynamically import DDSTools so we don't pay the bundle cost when its not needed.

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15879/merge/testResults/webgl2playwright/index.html

File: packages/tools/viewer-alpha/test/apps/web/index.html [hunk 1]
```diff
@@ -42,10 +42,10 @@
                 background-image: url("https://raw.githubusercontent.com/BabylonJS/Assets/master/textures/Checker_albedo.png");
             }
 
-             /* babylon-viewer-annotation:state(back-facing) {
+            /* babylon-viewer-annotation:state(back-facing) {
```
Comment (sebavan) [inline packages/tools/viewer-alpha/test/apps/web/index.html:45]: wondering how much comment we should keep or if we need some kind of modes to run it ?

**Review by sebavan — APPROVED** [review]

File: packages/tools/viewer-alpha/test/apps/web/index.html (hunk 1 above)
Comment (ryantrem) [reply packages/tools/viewer-alpha/test/apps/web/index.html:45]: There are buttons to test primary features, and for various other things I just have commented out code that is easy to uncomment for manual testing. I could add more buttons (or other UI) to the test app to toggle these things to test them, but I also don't want to over-invest in the test app...

Comment (sebavan) [reply packages/tools/viewer-alpha/test/apps/web/index.html:45]: totally agree on the over invest part

**Comment by sebavan** [comment]
you might need to merge master back in

**Pull request merged**